model/src/test/scala/uk/gov/ons/addressIndex/model/config/AddressIndexConfigTest.scala

```diff
@@ -18,7 +18,7 @@ object AddressIndexConfigTest {
 }
 
 class ElasticSearchConfigTest extends FlatSpec with Matchers {
-  it should "produce the expected default config" in {
+  ignore should "produce the expected default config" in {
```

Comment:
I wonder what is the purpose of this test, are we testing `PureConfig` ?
I remember you were saying that those tests are to prevent changes in in configuration without anybody noticing it, but we have code reviews for this kind of thing and having configuration parameter in 3 places (app.conf, default and tests) doesn't add anything to it in my opinion.